grails-app/migrations/0.8.x/changelog-2023-01-27-1015-add-synonym-classification-column.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="kchelstowski" id="270120231015-0">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="synonym" columnName="synonym_type_code" />
+            </not>
+        </preConditions>
+        <addColumn tableName="synonym">
+            <column name="synonym_type_code" type="VARCHAR(100)" defaultValue="DISPLAY_NAME"/>
```

Comment:
shouldn't defaultValue be `ALTERNATE_NAME` instead ? _(as specified in the ticket)_

Reply:
good catch - it was done first time round, before I consulted it with Manon and I've forgotten to change it then, thanks for viligance